src-tools/netmodel/netmodel.cpp

```diff
@@ -1101,6 +1103,11 @@ void asiotest_udpserv(std::vector<std::string> options) {
 	const bool cfg_tuntap_use_real_tun = func_cmdline("tuntap_use_real"); // if true real tuntap is used
 	const bool cfg_tuntap_async = func_cmdline("tuntap_async");
 
+    const string cfg_stats_format = func_cmdline_str("stats_format", std::string(""));
+    _mark("stats_format configuration: \"" << cfg_stats_format);
+
```

Comment:
typo, closing quotation character is missing?

Or, maybe use brackets "[" and "]"and quotation.